lib/policies/oauth2/site.js

```diff
@@ -5,9 +5,11 @@ const path = require('path');
 
 module.exports.loginForm = (request, response) => response.render(path.join(__dirname, 'views/login'));
 
-module.exports.login = passport.authenticate('local', { successReturnToOrRedirect: '/', failureRedirect: '/login' });
+module.exports.login = passport.authenticate('local', { successReturnToOrRedirect: '/', failureRedirect: '/login', keepSessionInfo: true });
```

Comment:
I was going questioning wether we want to set this to true, however the previous functionality on the previous versions this was default behaviour, so it keeps the feature as is, all good!